languages/omise-ja.po

```diff
@@ -18,21 +18,21 @@ msgstr ""
 "Language: ja\n"
 
 #: includes/gateway/class-omise-payment-alipay.php:21
-msgid "Omise Alipay"
-msgstr "Omise アリペイ"
+msgid "Opn Payments Alipay"
+msgstr "Opn Payments アリペイ"
 
 #: includes/gateway/class-omise-payment-alipay.php:22
 msgid "Accept payment through Alipay"
-msgstr "Omiseを通してアリペイ決済を提供します。"
+msgstr "Opn Paymentsを通してアリペイ決済を提供します。"
 
 #: includes/gateway/class-omise-payment-alipay.php:42 includes/gateway/class-omise-payment-creditcard.php:56
 #: includes/gateway/class-omise-payment-internetbanking.php:48
 msgid "Enable/Disable"
 msgstr "有効/無効"
 
 #: includes/gateway/class-omise-payment-alipay.php:44
-msgid "Enable Omise Alipay Payment"
-msgstr "Omiseのアリペイ決済を有効化します。"
+msgid "Enable Opn Payments  Alipay Payment"
```

Comment:
```suggestion
msgid "Enable Opn Payments Alipay Payment"
```

Reply:
Removed the extra space. Thanks for noticing it.